dev-python/pyBigWig/pyBigWig-0.3.24.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+DISTUTILS_USE_PEP517=setuptools
+export SETUPTOOLS_SCM_PRETEND_VERSION=${PV}
+inherit distutils-r1
+
+DESCRIPTION="quick access to and creation of bigWig files"
+HOMEPAGE="https://github.com/deeptools/pyBigWig"
+SRC_URI="https://github.com/deeptools/pyBigWig/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+REPEND="
+	sys-libs/zlib:0=
+	sci-biology/libBigWig
+"
+DEPEND="${RDEPEND}"
+
+distutils_enable_tests pytest
+
+python_test() {
+	local pysite="${BUILD_DIR}"/install/${EPREFIX}/usr/lib/${EPYTHON}/site-packages
+	ln -s "${S}"/pyBigWigTest/test.bigBed "${pysite}"/pyBigWigTest || die
+	ln -s "${S}"/pyBigWigTest/test.bw "${pysite}"/pyBigWigTest || die
+	epytest pyBigWigTest/test.py
```

Comment:
I'm not sure if there is a better way than linking

Reply:
What about setting `PYTHONPATH`?

Reply:
The problem here is that:

1. The test accesses the file using `os.path.dirname(pyBigWig.__file__) + "/pyBigWigTest/test.bigBed"`
2. `pyBigWig` only exists as `.so` file in the build directory (without those files)
3. Setting `PYTHONPATH` doesn't change the file path construction of the former

Reply:
`DISTUTILS_IN_SOURCE_BUILD=1` might help here.

Reply:
Right, let's just keep it as it is then.